### PR TITLE
Add formula for ElasticSearch 1.5.2.

### PR DESCRIPTION
**Pull request by scraton**
AWS recently released [ElasticSearch Service](https://aws.amazon.com/elasticsearch-service/). The version they currently run is 1.5.2, so having this installable via homebrew would be nice. :)

### CODE REVIEW

**Comment by scraton** [comment]
Sorry for all the extra commits. Long time leacher, first time contributor. :sweat_smile:

File: elasticsearch15.rb [hunk 1]
```diff
@@ -0,0 +1,133 @@
+class Elasticsearch15 < Formula
+  desc "You know, for search..."
```
Comment (DomT4) [inline elasticsearch15.rb]: Let's stick to the Homebrew description here:

```ruby
desc "Distributed real-time search & analytics engine for the cloud"
```

File: elasticsearch15.rb [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  homepage "https://www.elastic.co/products/elasticsearch"
+  url "https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.5.2.tar.gz"
+  sha256 "efae7897367ac3fa8057d02fad31c72e215b6edef599b63e373d3ce0c1049a14"
+
+  bottle do
```
Comment (DomT4) [inline elasticsearch15.rb]: Don't need this block here.

File: elasticsearch15.rb (hunk 1 above)
Comment (scraton) [inline elasticsearch15.rb]: Even though it causes the build/validations to fail?

File: elasticsearch15.rb (hunk 2 above)
Comment (scraton) [inline elasticsearch15.rb]: Copy that. Will remove.

File: elasticsearch15.rb (hunk 1 above)
Comment (DomT4) [inline elasticsearch15.rb]: Shorten it a bit if necessary.

**Comment by scraton** [comment]
Applied the requested changes. :)

**Comment by petercm** [comment]
+1
Though I'd bet you'll need to squash those commits

**Comment by mikemcquaid** [comment]
Out of interest: are there specific breaking changes that mean you want to be on the identical version to AWS? We're currently rethinking the way we do Homebrew versions to being +/-2 versions from core.

**Comment by scraton** [comment]
@mikemcquaid I'm not aware of any specific incompatibilities between the1.5 and 1.7. But I felt it'd be wise to develop against what AWS has to avoid any unpleasant surprises.

If you need me to squash these commits and open a new PR, just let me know. :)

**Comment by mikemcquaid** [comment]
Unfortunately we've remove all versions below 1.7 as they are now unmaintained by upstream: https://github.com/Homebrew/homebrew-versions/pull/1068. Sorry!

**Pull request closed without merging**